.travis.yml

```diff
@@ -1,23 +1,69 @@
-sudo: required
+# File managed by pluginsync
+sudo: false
 language: go
 go:
-- 1.5.4
-- 1.6.2
+- 1.6.3
+- 1.7.1
 env:
   global:
-    - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/intelsdi-x/snap-plugin-collector-haproxy
+  - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
+  - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
   matrix:
-    - TEST=unit
+  - TEST_TYPE=small
+  - TEST_TYPE=build
```

Comment:
The small test is running the same test as unit and currently we don't have medium tests. When we add medium test to this plugin, we can update the test matrix by updating the .sync.yml as described here: https://github.com/intelsdi-x/snap-pluginsync/#travisyml


We only build for go 1.7.1 because we only need one set of binaries for s3. (see https://travis-ci.org/intelsdi-x/snap-plugin-collector-haproxy/jobs/161439442) We choose to build with latest go version, but test with latest and previous release, which is why go 1.6.3 is excluded from the build matrix.